fvm/src/kernel/default.rs

```diff
@@ -816,6 +816,10 @@ where
                 .price_list()
                 .on_get_randomness(entropy.len()),
         )?;
+
+        self.call_manager
+            .charge_gas(self.call_manager.price_list().on_extern())?;
+
```

Comment:
Nit: This repeated pattern makes me think that we should have a capability to either charge for both charges at once or the capability to combine charges into one but it isn't something that needs to get done.

It also shouldn't be consensus critical if we want to add this capability in future.

Reply:
Agree. I started hacking on a const generics version of `charge_gas_n` that took an array and applied all charges sequentially. A few months ago I had also proposed a closure-based version of `charge_gas` that took the price list as an argument. That would also remove some pedanticness. However, we can explore these APIs later; right now it doesn't hurt having two explicit charges.